### PR TITLE
Fix ut

### PR DESCRIPTION
**Pull request by shargon**
Close https://github.com/neo-project/neo/issues/3112

### CODE REVIEW

File: src/Neo/NeoSystem.cs [hunk 1]
```diff
@@ -211,16 +208,6 @@ public void EnsureStopped(IActorRef actor)
             inbox.Receive(TimeSpan.FromMinutes(5));
         }
 
-        /// <summary>
-        /// Loads an <see cref="IStore"/> at the specified path.
-        /// </summary>
-        /// <param name="path">The path of the storage.</param>
-        /// <returns>The loaded <see cref="IStore"/>.</returns>
-        public IStore LoadStore(string path)
```
Comment (Jim8y) [inline src/Neo/NeoSystem.cs:219]: This is public, maybe we should not just delete it like this.

Comment (shargon) [reply src/Neo/NeoSystem.cs:219]: Is `Load` but it only returns the `StoreFactory` result (that is public too)

Comment (shargon) [reply src/Neo/NeoSystem.cs:219]: mmm maybe it's used by plugis for create a new one, let me check

**Comment by Jim8y** [comment]
LGTM

**Comment by Jim8y** [comment]
@superboyiii  need you here

**Comment by shargon** [comment]
> @superboyiii need you here

I prefer @superboyiii in #3082 :) here is fixed

**Comment by Jim8y** [comment]
Is that possible that you make the tiny methods in MemoryStore aggressiveinline here?

**Comment by superboyiii** [comment]
Will test this when it's updated.

**Comment by Jim8y** [comment]
@shargon needs update

**Review by Jim8y — APPROVED** [review]

**Comment by Jim8y** [comment]
@cschuchardt88 @NGDAdmin

**Pull request merged**